Reference/Mapping/index-vpre8.1.md

```diff
@@ -0,0 +1,7 @@
+---
+versionTo: 8.1.0
+---
+
+# Mapping in Umbraco
+
+Prior to Umbraco 8.1 mapping was handled by an external dependency AutoMapper, to improve startup times and reduce external dependencies it has been replaced by UmbracoMapper in 8.1.0+.
```

Comment:
replaced in 8.1.0+ by a simpler Umbraco specific implementation called UmbracoMapper.